packages/adwaita-icon-theme/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.gnome.org
 TERMUX_PKG_DESCRIPTION="GNOME standard icons"
 TERMUX_PKG_LICENSE="LGPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=40.1.1
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=41.0
+TERMUX_PKG_REVISION=2
```

Comment:
On version upgrades `TERMUX_PKG_REVISION` should be unset. It should be incremented only when doing rebuild within the same version.

Reply:
Set it to 0 or what else?

Reply:
Unset means remove entire line with `TERMUX_PKG_REVISION`.